common/workunit/workunit.cpp

```diff
@@ -14137,23 +14137,31 @@ KeepK8sJobs translateKeepJobs(const char *keepJob)
     return KeepK8sJobs::none;
 }
 
+// NB: will fire an exception if command fails (returns non-zero exit code)
+static void runKubectlCommand(StringBuffer *output, const char *title, const char *cmd, const char *input, bool logOutput, bool logErrors)
+{
+    StringBuffer _output, error;
+    if (!output)
+        output = &_output;
+    bool ret = runExternalCommand(title, *output, error, cmd, input);
+    if (logOutput && output->length())
```

Comment:
I wonder if should be (logOutput || traceLevel > 5) or similar...

Reply:
could just leave it down to jlog log levels.
It was recently reduced to 80 - reducing any lower would prevent DBGLOG /debugInfo logging from being issued.
I don't think we can do that yet, until all DBGLOG logging is looked at, there's bound to be some that should really be at a higher level.

But I think we can change this logging here to use a higher (than default) level, so it is not output unless component jlog level is raised, i.e. ExtraneousMsgThreshold